test/e2e/e2e_test.go

```diff
@@ -72,4 +74,39 @@ var _ = Describe("LWS Operator", Ordered, func() {
 			return nil
 		}, 5*time.Minute, 5*time.Second).Should(Succeed(), "available condition is not found")
 	})
+	It("Verifying operand pod deleted and recovery", func() {
+		// Delete Operand Pod
+		ctx := context.TODO()
+		pods, err := clients.KubeClient.CoreV1().Pods("openshift-lws-operator").List(ctx, metav1.ListOptions{
+			LabelSelector: "control-plane=controller-manager",
+		})
+		if err != nil {
+			klog.Errorf("Failed to list pods: %v", err)
+		}
+		for _, pod := range pods.Items {
+			err = clients.KubeClient.CoreV1().Pods("openshift-lws-operator").Delete(ctx, pod.Name, metav1.DeleteOptions{})
```

Comment:
Maybe instead of using a loop, we can delete by passing labelSelector. WDYT?

Reply:
Updated